Cargo.toml

```diff
@@ -56,6 +56,7 @@ semver = { version = "1.0.24", optional = true }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 uuid = { version = "1.13.1", default-features = false, features = ["js"] }
+getrandom = { version = "0.3.3", features = ["wasm_js"] }
```

Comment:
I'm a bit concerned about this warning:

> WARNING: It is highly recommended to enable the wasm_js feature only for binary crates and tests, i.e. avoid unconditionally enabling it in library crates.

I tried changing this to only apply for `dev-dependencies`, but that didn't work. The compilation failed while the library was building, before it ever got around to building the tests.

I suppose we're not enabling this feature **_unconditionally_**; we're only enabling it for wasm32 architectures. However, I'm not sure if there are situations where someone might target a wasm32 architecture and not want this feature enabled.

Reply:
> We could also not change the deps but just hack it so CI passes.

My worry at that point is downstream users will have to go through the trouble of figuring all of this out. In fact it's not clear to me whether builds of downstream crates will be able to inherit the `rustflags` from `.cargo/config.toml`. I guess we'll need to test that out later. For now we can just move forward with this PR to unblock CI.

Reply:
One way is to use a custom build script that sets the cfg, but I think there are reasons why `getrandom` does not do that.